include/xtensor/xchunked_array.hpp

```diff
@@ -155,6 +141,49 @@ namespace xt
             }
         }
 
+        template <class E, class S>
+        xchunked_array(const xexpression<E>& e, S chunk_shape)
```

Comment:
You need to keep the constructor that accepts an expression only, otherwise you cannot write things such as

```cpp
xchunked_array<chunk_type> a = b + c;
```

Reply:
Do you mean we cannot have both? The previous one (`xchunked_array(const xexpression<E>& e)`) is still present.

Reply:
Ha sorry I missed it ;)